### ISSUE TITLE
Vulkan API requring vulkan headers

### ISSUE THREAD

**Issue by JRBurn777** [Dav1dde/glad#405]
When generating [Glad for Vulkan](https://gen.glad.sh/), the VK_EXT_video_* and VK_KHR_video_* extensions require the [vk_video Vulkan headers](https://github.com/KhronosGroup/Vulkan-Headers/tree/main/include/vk_video).

```
include/glad/vulkan.h|7243 col 2| #include "vk_video/vulkan_video_codec_h264std.h"
include/glad/vulkan.h|7246 col 2| #include "vk_video/vulkan_video_codec_h264std_decode.h"
include/glad/vulkan.h|7296 col 2| #include "vk_video/vulkan_video_codec_h265std.h"
include/glad/vulkan.h|7302 col 2| #include "vk_video/vulkan_video_codec_h265std_decode.h"
include/glad/vulkan.h|7372 col 2| #include "vk_video/vulkan_video_codec_h264std_encode.h"
include/glad/vulkan.h|7489 col 2| #include "vk_video/vulkan_video_codec_h265std_encode.h"
```
The last two in that list are protected by "VK_ENABLE_BETA_EXTENSIONS" which I do not define, but the initial four includes cause build issues without additionally pulling in the vk_video headers.

I can avoid the issue by removing all VK_EXT_video_* and VK_KHR_video_* extensions before generating.

Is this intentional behavior?

**_Generator Options to reproduce_**
APIs:

- Vulkan=1.3

Extensions:

- "Add All"

Options:

- ALIAS = True
- DEBUG = False
- HEADER_ONLY = False
- LOADER = False
- MX = True
- ON_DEMAND = False

**Comment by JRBurn777** [comment]
As an update, with the addition of video headers in this patch: https://github.com/Dav1dde/glad/commit/71f30638529eedfad3e9c47b0e6940dd1a7cf727

Generating for Vulkan 1.3 and all extensions almost works.
If I pull down [vulkan_video_codecs_common.h](https://github.com/KhronosGroup/Vulkan-Headers/blob/main/include/vk_video/vulkan_video_codecs_common.h) and place it with the other vk_video headers, I'm able to build and run.

**Comment by Dav1dde** [comment]
Fixed with 7e52c19d0991b237979eed67e4115b8d182a6c6c
